journal/modernized/27-chapter-24.adoc

```diff
@@ -590,9 +591,9 @@ and lead some to inquire into the cause.
 accompanying a translated copy of a note, with the original, from Mauiui,
 our pilot through the reef, as follows:--
 
-'`As I know not the names of you two gentlemen, I address you thus generally.
+As I know not the names of you two gentlemen, I address you thus generally.
```

Comment:
I've come across this in the next chapters as well. It seems that a single quote sets off a quoted written note or something similar that he's relaying. These single quotes are in the pdf, so I'm not sure if I should leave them or not, especially since I've seen them in upcoming chapters as well. There aren't any end quotes, but the sections are offset so it stands out enough that I don't think it's necessary to keep the single quote in the modernized at least. Also, this is my same correction on line 596 too.

Reply:
Yeah, I think removing is good here. Technically, I think we should wrap this section with some more asciidoc indicating that it's an "embedded letter", but that's not something you know how to do (yet!), so leave it for now. I made a note so that I or someone else won't forget to come back and add that later.